roles/openshift_common/tasks/atomic_docker_check.yml

```diff
@@ -0,0 +1,17 @@
+---
+# Fail early if Atomic and old version of Docker
+- block:
+
+  - name: Determine available Docker
+    shell: "rpm -q --queryformat '---\ncurr_version: %{VERSION}\navail_version: \n' docker"
```

Comment:
I pulled this from the docker upgrade steps. I'll fix that up to be easier to read.